ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/store/state.proto

```diff
@@ -54,12 +54,10 @@ message DamlCommandDedupKey {
 
 message DamlCommandDedupValue {
   reserved 1; // was record_time
+  reserved 3; // was submission_time
```

Comment:
If we introduced the `submission_time` field after SDK 1.17.x (that is, within the timeframe of 1.18), then there's no need to put it as "reserved" field, but you can just remove it.

Reply:
Within a release, there's no need to preserve backwards compatibility.

Reply:
The field was introduced after 1.17.

If this is considered safe, and it's safe to break compatibility between 1.18 RC1 and 1.18 RC2 then there's actually no need for https://github.com/digital-asset/daml/pull/11781